config/samples/craystack_lustrefilesystem.yaml

```diff
@@ -3,12 +3,20 @@ kind: Namespace
 metadata:
   name: nnf-daffy-lustre
 ---
-apiVersion: cray.hpe.com/v1alpha1
+apiVersion: lus.cray.hpe.com/v1alpha1
 kind: LustreFileSystem
 metadata:
   name: w0-lustre-fs
   namespace: nnf-daffy-lustre
 spec:
   name: w0
-  mgsNid: rabbit-03@tcp
+  mgsNids: rabbit-03@tcp
   mountRoot: /lus/w0
+  namespaces:
+    nnf-daffy-lustre:
+      modes:
+        - ReadWriteMany
```

Comment:
Are these testing changes or necessary updates?

Reply:
These are necessary, if you want to use these samples for manual testing.